src/azure/cli/commands/parameters.py

```diff
@@ -73,6 +73,19 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 def get_enum_choices(enum_type):
     return [x.value for x in enum_type]
 
+class IgnoreAction(argparse.Action): # pylint: disable=too-few-public-methods
+    def __call__(self, parser, namespace, values, option_string=None):
+        raise argparse.ArgumentError(None, 'unrecognized argument: {} {}'.format(
+            option_string, values or ''))
+
+# CONSTANTS
+
+IGNORE_TYPE = CliArgumentType(
```

Comment:
works for me, but I would remove the # CONSTANTS line since these aren't truly constants 

---
In reply to: [76153082](https://github.com/Azure/azure-cli/pull/744#discussion_r76153082) [](ancestors = 76153082,76152207,76151727,76151354)